matrices/views.py

```diff
@@ -34,4 +34,45 @@ def matrice_ex1(request):
                 maior = matrices[i][j]
                 position = i, j
 
-    return render(request, 'matrices/matrice_ex1.html', {'matrices': matrices, 'maior': maior, 'position': position})
\ No newline at end of file
+    return render(request, 'matrices/matrice_ex1.html', {'matrices': matrices, 'maior': maior, 'position': position})
+
+
+def matrice_ex5(request):
+    value2 = []
+    matrices = []
+    error2 = None
+    value2_int = None
+
+    if request.method == 'POST':
+        value = request.POST.get('value1').split(',')
+
+        if len(value) != 5:
+            error = 'Plaese write just 10 values.'
+            return render(request, 'matrices/matrice_ex5.html', {'error': error})
+
+        for i in value:
+            int_i = int(i)
+            if int_i < 10:
+                error = 'Plaese write values grater than 9.'
+                return render(request, 'matrices/matrice_ex5.html', {'error': error})
+
+            matrices.append([random.randint(1, int_i) for _ in range(5)])
+
+        value2 = request.POST.get('value2').split(',')
+
+        value2_int = int(value2[0])
+        positions = []
+        for i in range(0, len(matrices)):
```

Comment:
A gente já falou desse problema 
Essa não é a solução correta 
A solução é 1 (um apenas 1 ) loop em cada elemento da matris e em cada iteração checar se o número e o q estamos procurando

Reply:
here had six loops, i fixed and used just two loops.